src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/CodeActionOptions.cs

```diff
@@ -74,4 +81,43 @@ public CodeActionOptions()
 #endif
 
     internal delegate CodeActionOptions CodeActionOptionsProvider(HostLanguageServices languageService);
+
+    internal static class CodeActionOptionsProviders
```

Comment:
Defines extensions methods that make fetching options from Document and converting between providers convenient.